src/System.Private.ServiceModel/src/System/IdentityModel/Selectors/SecurityTokenSerializer.cs

```diff
@@ -207,41 +208,5 @@ public virtual bool CanReadKeyIdentifierCore(XmlDictionaryReader reader)
 
             public abstract void WriteKeyIdentifierCore(XmlDictionaryWriter writer, SecurityKeyIdentifier keyIdentifier);
         }
-
-        internal abstract class TokenEntry
```

Comment:
Is this the right way around to do things? WCF originally had all these classes internally, and then IdentityModel was created and duplicated there and not all the WCF versions were eliminated. If IdentityModel were to come back, then we would likely want to do another sweep and remove them again.

---
In reply to: [78077394](https://github.com/dotnet/wcf/pull/1515#discussion_r78077394) [](ancestors = 78077394)